File: C++/Valid Palindrome.cpp
#include <algorithm>
class Solution {
public:
    bool isPalindrome(string s) {
        
        string m = "";

        for (uint8_t x : s) {
            
            if (x < 48 || (x > 57 && x < 65) || (x > 90 && x < 97) || x > 122) continue;
            if (x >= 65 && x <= 90) x += 32; //* lower case

            m += (char) x;

        }

        string n = m;
        std::reverse(n.begin(), n.end());
        return (m == n);

    }
};

File: python/Valid Palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        
        m = ""
        s = s.lower().strip()
        
        for x in s:
            if x == "" or not x.isalnum():
                continue
            m += x

        return m[::-1] == m
